src/controller/ordersControllers.js

```diff
@@ -1,37 +1,40 @@
 import Joi from 'joi';
 import orders from '../models/orders';
 
+export function getAllOrders (req, res) {
+	res.status(200).json({status: 'Success', Orders: orders});
+}
 
 export function postOrder (req, res) {
-    const {error} = validateOrders(req.body);
+	const {error} = validateOrders(req.body);
     
-    if(error) return res.status(400).json({status: 'success', message: error.details[0].message});
+	if(error) return res.status(400).json({status: 'Failed to create order', message: error.details[0].message});
 
-    const order = {
-        id: orders.length + 1,
-        type: req.body.type,
-        name: req.body.name,
-        orderDate: req.body.orderDate,
-        expires: req.body.expires,
-        quantities: req.body.quantities,
-        unitPrice: Math.floor( Math.random() + 200),
-        total: 'N' + req.body.quantities * (Math.floor( Math.random() + 200))
-    };
+	const order = {
+		id: orders.length + 1,
+		type: req.body.type,
+		name: req.body.name,
+		orderDate: req.body.orderDate,
+		expires: req.body.expires,
+		quantities: req.body.quantities,
+		unitPrice: Math.floor( Math.random() + 200),
+		total: 'N' + req.body.quantities * (Math.floor( Math.random() + 200))
+	};
 
-    orders.push(order);
+	orders.push(order);
 
-    res.json(order);
+	res.json({justAdded: order, message: 'order Created'});
 }
 
 function validateOrders(order) {
```

Comment:
#### Critical
Same feedback on #11

Reply:
Thanks so much, it has been taken care of